sql/core/src/test/scala/org/apache/spark/sql/execution/joins/BroadcastJoinSuite.scala

```diff
@@ -425,6 +425,29 @@ abstract class BroadcastJoinSuiteBase extends QueryTest with SQLTestUtils
     }
   }
 
+  test("SPARK-33933: AQE broadcast should not timeout with slow map tasks") {
```

Comment:
how long does this test run?

Reply:
About 10+ sec, given the master conf "local-cluster[2,1,1024]"

Reply:
is it possible to make it a bit faster? e.g. reduce the number of map task, reduce the timeout value, etc.

Reply:
I will try to reduce to about 3~5 sec